bokehjs/src/lib/models/glyphs/circle.ts

```diff
@@ -17,11 +17,13 @@ export interface CircleData extends XYGlyphData {
   _angle: NumberArray
   _size: NumberArray
   _radius?: NumberArray
+  _hit_radius?:NumberArray
 
   sradius: NumberArray
 
   max_size: number
   max_radius: number
+  max_hit_radius: number
```

Comment:
I don't understand what `max_hit_radius` is intended for,  and I think it is unnecessary. Maybe because of the `ScreenDistanceSpec` used? When that becomes `Float` there is definitely no purpose to this. Same for `_hit_radius`. You can (should) just use `this.model.hit_radius` wherever the  value is needed.